### PR TITLE
[WIP] Update downloads page for 3.10.0

### PR DESCRIPTION
**Pull request by brianlheim**
- remove sc3-plugins from windows downloads (link is on page)
- remove versions before 3.7.2 from download links
- remove info about 3.7.2 vs 3.6.6 (3.6.6 no longer linked on page)
- add 3.10.0 links
- correct spelling of sc3-plugins
- make external links open in new tab/window
- redirect sc3-plugins link to new site

several of these points are up for discussion. i just thought the
windows release links were getting crowded with links to sc3-plugins,
and there's a link to it below anyway.

### CODE REVIEW

**Comment by brianlheim** [comment]
still need to write release blurb for 3.10. assuming we'll use the same text for 3.10.0. @snappizz - do you have anything? i can help write/review

File: download.md [hunk 1]
```diff
@@ -7,23 +7,19 @@ group: navigation
...
+    The sc3-plugins are extension plugins for the SuperCollider audio synthesis server. These third-party plugins provide additional synthesis, analysis, and other capabilities for the sound server.
 
-- [Quarks](https://github.com/supercollider-quarks/quarks)
+- <a href="https://github.com/supercollider-quarks/quarks" target="_blank">Quarks</a>
 
     Quarks are packages containing classes, extension methods, documentation and UGens. The integrated Quarks package system is used to download and manage these packages on your system.
-
-    **[SuperCollider 3.7.0 or later]** Quarks can be installed by graphical user interface, by GitHub URL, or by manually placing files in SuperCollider's Extensions folder. For more information, see the [SuperCollider Quarks](https://github.com/supercollider-quarks/quarks) project page.
-
-    **[SuperCollider 3.6.6 and earlier]** See the old [SVN Quarks repository](http://sourceforge.net/p/quarks/code/HEAD/tree/) and consult the help file included with your version of SuperCollider.
```
Comment (patrickdupuis) [inline download.md]: We should discuss removing these on our next call. They don't need to be removed as part of this PR.

File: download.md [hunk 2]
```diff
@@ -7,23 +7,19 @@ group: navigation
 {% include download.md %}
 
 ### Releases
-[SuperCollider Releases](https://github.com/supercollider/supercollider/releases) (including Stable Releases, Betas, and Release Candidates)
+<a href="https://github.com/supercollider/supercollider/releases" target="_blank">SuperCollider Releases</a> (including Stable Releases, Betas, and Release Candidates)
```
Comment (patrickdupuis) [inline download.md]: Why change these from markdown links to anchors? `_includes/download.md` is an exception. The rest of the website uses markdown links IIRC.

Comment (patrickdupuis) [reply download.md]: oh... `make external links open in new tab/window`. This is the only way to accomplish this?

Comment (patrickdupuis) [reply download.md]: There's this: https://github.com/keithmifsud/jekyll-target-blank
I don't know if it's worth the hassle. I'm ok with these changes.

File: download.md (hunk 1 above)
Comment (patrickdupuis) [reply download.md]: I've changed my mind. Let's get rid of as much text as possible before the work on the website redesign.

File: download.md (hunk 2 above)
Comment (brianlheim) [reply download.md]: yes this is the only way i could find. looks like that does this to all external links, but i really don't see the point, it's not too bad to just manually insert them (with a vim macro takes about 1 ms)

**Comment by brianlheim** [comment]
Updated, this is the announcement text i'd like to use for all our announcements. as soon as this is merged we can announce it everywhere :)

**Pull request merged**